packages/component-library/src/components/table-and-list/mt-data-table/mt-data-table.vue

```diff
@@ -2729,11 +2723,12 @@ table.is--dragging-inside {
   &.mt-data-table__table-head-add-column-indicator {
     cursor: pointer;
     z-index: 10;
-    background-color: $color-white;
 
     .mt-icon {
       display: block;
       color: $color-darkgray-200;
+      background: #fff;
```

Comment:
```suggestion
      color: var(--color-text-secondary-default);
      background: var(--color-icon-inverted-default);
```

Maybe 🤷‍♂️ Depends on where this is applied 😄